### PR TITLE
fix: microblocks + hyperchain adjustments 

### PR DESCRIPTION
**Pull request by lgalabru**
This PR:
- Adds the ability to use microblocks in deployment plans
- Displays microblocks in the clarinet terminal ui
- Fixes a bug when setting microblock's block identifiers + parent block identifiers 
- Fixes a bug in the hyperchain integration where the contract_id of the hyperchain contract was not using the remapped version. 
- Fixes a bug where clarinet integrate would take ages to terminate itself.

### CODE REVIEW

File: components/clarinet-files/src/network_manifest.rs [hunk 1]
```diff
@@ -422,7 +433,7 @@ impl NetworkManifest {
                     .unwrap_or("devnet".to_string()),
                 bitcoin_controller_block_time: devnet_config
                     .bitcoin_controller_block_time
-                    .unwrap_or(30_000),
+                    .unwrap_or(90_000),
```
Comment (obycode) [inline components/clarinet-files/src/network_manifest.rs:436]: Oh bummer, did it need to be slowed down to function properly?

File: src/integrate/ui/mod.rs [hunk 2]
```diff
@@ -97,30 +99,52 @@ pub fn start_ui(
...
+                        for i in indices_to_remove {
+                            app.mempool.items.remove(i);
+                        }
+                        for block_update in update.new_microblocks.into_iter() {
+                            app.display_microblock(block_update);
+                        }
                     }
-                } else {
-                    // TODO(lgalabru)
+                    _ => {} // handle display on re-org, theorically unreachable in context of devnet
```
Comment (obycode) [inline src/integrate/ui/mod.rs:147]:
```suggestion
                    _ => {} // handle display on re-org, theoretically unreachable in context of devnet
```

File: vendor/orchestra-event-observer/src/indexer/stacks/blocks_pool.rs [hunk 3]
```diff
@@ -670,7 +691,7 @@ impl StacksBlockPool {
                     Some(block) => block.clone(),
                     None => {
                         error!(
-                            "unable to retrive block {} from block store",
+                            "unable to retrive Stacks {} from block store",
```
Comment (obycode) [inline vendor/orchestra-event-observer/src/indexer/stacks/blocks_pool.rs:694]:
```suggestion
                            "unable to retrieve Stacks {} from block store",
```

**Review by obycode — DISMISSED** [review]
Looks good!

File: components/clarinet-files/src/network_manifest.rs (hunk 1 above)
Comment (lgalabru) [reply components/clarinet-files/src/network_manifest.rs:436]: No, 30s still works as expected, but now that microblocks are correctly handled and we have visual feedback for it in the terminal UI, using a longer block time might be more faithful to what will happen on Mainnet.

**Review by obycode — APPROVED** [review]

**Pull request merged**